### ISSUE TITLE
object-curly-spacing throws false positives for export statements

### ISSUE THREAD

**Issue by jonathanewerner** [eslint/eslint#3876]
```
export Box from './Box.jsx';
export Button from './Button.jsx';
export GoogleAnalytics from './GoogleAnalytics.jsx';
export GoogleMap from './GoogleMap.jsx';
```

This throws
```
  12:18  warning  There should be no space before 'from'   object-curly-spacing
  13:7   warning  There should be no space after 'export'  object-curly-spacing
  13:16  warning  There should be no space before 'from'   object-curly-spacing
  14:7   warning  There should be no space after 'export'  object-curly-spacing
  14:16  warning  There should be no space before 'from'   object-curly-spacing
  15:7   warning  There should be no space after 'export'  object-curly-spacing
  15:13  warning  There should be no space before 'from'   object-curly-spacing
```
... which seems like a false-positive to me?

---
eslint-version: v1.5.0
relevant rule in .eslintrc: `"object-curly-spacing": [1, "never"],`

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by michaelficarra** [comment]
That's not valid ECMAScript. Put curlies around your export specifier list.

**Comment by michaelficarra** [comment]
What parsing are you using that accepted that as a program? You should report a bug there as well.

**Comment by jonathanewerner** [comment]
babel. I use `babel-eslint` with `eslint-loader` via webpack. The code compiles fine.

**Comment by michaelficarra** [comment]
They must be implementing a proposal or just have a bug. Either way, it shouldn't work, and this is not a bug in eslint.

**Comment by mysticatea** [comment]
Thanks for reporting @jonathanewerner.

I agree with @michaelficarra.
That syntax is not supported in ECMAScript yet.
I recommend using [eslint-plugin-babel](https://github.com/babel/eslint-plugin-babel) since you are using the `babel-eslint` parser.
